package.json

```diff
@@ -4,11 +4,14 @@
   "description": "Movio's TodoMVC stack",
   "main": "index.js",
   "scripts": {
-    "start": "webpack-dev-server --hot --inline --progress --colors",
+    "start": "webpack-dev-server --hot --inline --progress --colors --host 0.0.0.0",
     "build": "webpack --progress --colors",
     "test": "standard-format -w",
-    "lint": "eslint",
-    "format": "eslint"
+    "lint": "eslint src",
+    "format": "eslint src --fix"
+  },
+  "eslintConfig": {
```

Comment:
Sure can. I prefer it in the package - I have no idea why.
Do we prefer these kind of configs, eslint, babel etc in dot files or package? (I think I know the answer ...)